### DISCUSSION TITLE
Cannot access my projects

### DISCUSSION THREAD

**Discussion by nitrogen404** [supabase/supabase#8519] [Questions]
I'm using supabase for over 2 months now. Everything was working fine until this morning when I couldn't see any of my personal and organisation projects. I got logged out and when I logged in again, I couldn't see any of my personal and organisation projects. I tried again to log in after a few attempts I got to this.

![image](https://user-images.githubusercontent.com/76176835/185554047-db88a4e2-cc63-49cb-b2b1-d4e9031fc6d9.png)

Even after reauthorizing, I couldn't see my projects and after a new invite from my organisation, projects weren't visible.

**Comment by J0** [comment]
Hey @nitrogen404,

 We are sorry to hear that you are facing issues with loading projects -- could we trouble you to shoot us an email at support@supabase.io along with your project ref?

Thanks!
